### DISCUSSION TITLE
guide-R/analyses/regression-lineaire

### DISCUSSION THREAD

**Discussion by giscus** [larmarange/guide-R#45] [Forum]
# guide-R/analyses/regression-lineaire

https://larmarange.github.io/guide-R/analyses/regression-lineaire.html

**Comment by Lakus-beep** [comment]
Bonjour j'aurais une questions, j'espère que j'aurais une réponse. pour la partie 22.3, que veut dire la p-value ? Est ce qu'elle permet de dire que les résultats sont cohérent ? C'est à dire qu'étant inférieur à 0.05, on peut affirmer que la largeur des pétales dépend fortement et positivement de la longueur de pétale et de longueur de sépale. Je vous remercie de votre aide !

**Reply by larmarange** [reply]
Oui, avec une p valeur faible, il est possible de dire, qu'après ajustement sur les autres variables du modèle (i.e. toutes choses égales par ailleurs), il y a une corrélation positive et significative entre la largeur et la longueur d'un pétale. 
Cdlt

**Comment by boukirilatifa-hub** [comment]
Bonjour! Je vous écris afin de clarifier ma confusion 1/1 à la régression logistique. Je travail sur un cas concret à travers un questionnaire contenant 108 questions et un modèle économétrique ou la variable endogène prend deux modalités( Oui/Non).Ma question est la suivante: Comment effectuer une analyse d'un modèle logit tout en prenant en compte toutes ces questions. Je vous remercie par avance de votre assistance.

**Reply by larmarange** [reply]
Vous ne pouvez pas introduire en bloc vos 108 questions.
Vous devez impérativement faire un point de littérature en amont pour sélectionner les variables pertinentes à intégrer dans votre modèle.

Dans un second temps, vous pouvez améliorer votre modèle en le réduisant, voir : https://larmarange.github.io/guide-R/analyses/selection-modele-pas-a-pas.html
